dbfit-java/core/src/main/java/dbfit/util/FitNesseTestHost.java

```diff
@@ -9,6 +10,7 @@
 
     static {
         TypeAdapter.registerParseDelegate(BigDecimal.class, BigDecimalParseDelegate.class);
+        TypeAdapter.registerParseDelegate(BigInteger.class, BigIntegerParseDelegate.class);
         TypeAdapter.registerParseDelegate(java.sql.Date.class, SqlDateParseDelegate.class);
         TypeAdapter.registerParseDelegate(java.sql.Timestamp.class, SqlTimestampParseDelegate.class);
         TypeAdapter.registerParseDelegate(java.sql.Time.class, SqlTimeParseDelegate.class);
```

Comment:
For my own understanding of how this works: when do we actually need to create parse delegates? Only when the class doesn't automatically provide a useful 'parse' method?